src/content-helper/editor-sidebar/performance-details/component-panel-categories.tsx

```diff
@@ -89,42 +89,43 @@ export const PerformanceCategoriesPanel = ( {
 					</SelectControl>
 				</div>
 			) }
-			{
-				( isLoading ? (
-					<div className="parsely-spinner-wrapper" data-testid="parsely-spinner-wrapper">
-						<Spinner />
-					</div>
-				) : (
-					<div>
-						<div className="multi-percentage-bar">{
-							Object.entries( data.referrers.types ).map( ( [ key, value ] ) => {
-								const ariaLabel = sprintf(
-									/* translators: 1: Referrer type, 2: Percentage value, %%: Escaped percent sign */
-									__( '%1$s: %2$s%%', 'wp-parsely' ),
-									getKeyTitle( key ), value.viewsPercentage
-								);
+			{ isLoading ? (
+				<div className="parsely-spinner-wrapper" data-testid="parsely-spinner-wrapper">
+					<Spinner />
+				</div>
+			) : (
+				<div>
+					<div className="multi-percentage-bar">
+						{ Object.entries( data.referrers.types ).map( ( [ key, value ] ) => {
+							const ariaLabel = sprintf(
+								/* translators: 1: Referrer type, 2: Percentage value, %%: Escaped percent sign */
+								__( '%1$s: %2$s%%', 'wp-parsely' ),
+								getKeyTitle( key ),
+								value.viewsPercentage,
+							);
 
-								return (
-									<div aria-label={ ariaLabel }
-										className={ 'bar-fill ' + key } key={ key }
-										style={ { width: value.viewsPercentage + '%' } }>
-									</div>
-								);
-							} ) }
-						</div>
-						<div className="percentage-bar-labels">
-							{
-								Object.entries( data.referrers.types ).map( ( [ key, value ] ) => (
-									<div className={ 'single-label ' + key } key={ key }>
-										<div className={ 'label-color ' + key }></div>
-										<div className="label-text">{ getKeyTitle( key ) }</div>
-										<div className="label-value">{ formatToImpreciseNumber( value.views ) }</div>
-									</div>
-								) )
-							}
-						</div>
+							return (
+								<Tooltip text={ `${ getKeyTitle( key ) } - ${ value.viewsPercentage }%` } delay={ 150 } key={ key }>
```

Comment:
Some languages put a space before the percent sign, or put it on the left of the value. So let's use `__()` for the `text` property.